tasks.php

```diff
@@ -1916,4 +1975,13 @@ function title_string_for_task($pre_task)
     return sprintf("Task #%d: %s", $pre_task->task_id, $pre_task->task_summary);
 }
 
+// Convert any URLs into a clickable link in a string
+function make_urls_links($string)
+{
+    // from https://stackoverflow.com/questions/1960461/convert-plain-text-urls-into-html-hyperlinks-in-php
+    $url = '/(http|https|ftp|ftps)\:\/\/[a-zA-Z0-9\-\.]+\.[a-zA-Z]{2,3}(\/\S*)?/';
+    $string = preg_replace($url, '<a href="$0" target="_blank">$0</a>', $string);
```

Comment:
Should we be using `target="_blank"` here?

Reply:
Nope, I will remove it.